### DISCUSSION TITLE
Load images dynamically.

### DISCUSSION THREAD

**Discussion by TomSchimansky** [r0x0r/pywebview#1208] [Q&A] [answered]
When I create a window like this and try to add an image its not working:
```python
import webview

def insert_image():
    path = "img.jpg"
    window.evaluate_js(f"img = document.createElement('img');" +\
                       f"img.src = '{path}';" +\
                       f"document.getElementById('body').appendChild(img);\n")

window = webview.create_window(html=f"<html><head></head><body></body></html>")
webview.start(insert_image)
```
How does the path have to look like? the image is in the same folder as the python script. It's also not working with the full path. Also I don't want to use an extra html file for creating the basic window, I want to work with a single .py file and add the content dynamically.

**Comment by r0x0r** [comment, marked as answer]
There is no access to file system when you load HTML dynamically. You have several choices
1) Use a local HTTP server. Then paths will be resolved relative to the script location.
2) Load image as a base64 string
3) Load image from a remote server
